XouDevSpec/Classes/Model/MangaFaveStruct.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  MangaFaveStruct.swift
+//  XouDevSpec
+//
+//  Created by Xander Schoeman on 2020/03/26.
+//
+
+import Foundation
+
+    public struct faveMangaInfo: Decodable {
+        public var request_hash: String?
+        public var request_cached: Bool?
+        public var request_cache_expiry: Int?
+        public var results = [faveMangaDetails]()
+        public var last_page : Int?
+
+        public init() {
+            request_hash.self = ""
+            request_cached.self = true
+            request_cache_expiry.self = 0
+            last_page.self = 0
```

Comment:
you can set the default values for these in the instantiation, then you won't have to specify an init()